ntc_templates/templates/arista_eos_show_ip_mroute_vrf_all_detail.textfsm

```diff
@@ -0,0 +1,43 @@
+Value Filldown VRF (\S+)
+Value Filldown MULTICAST_GROUP (\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})
+Value Required RPF_SOURCE (\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})
+Value RP (\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})
+Value UPTIME (\d+d\d+h|\d+:\d+:\d+)
+Value FLAGS (\S+)
+Value INCOMING_INTERFACE (\S+)
+Value UPSTREAM_JOINED_STATE (\S+)
+Value UPSTREAM_RPT_JOINED_STATE (\S+)
+Value List OUTGOING_INTERFACE_LIST (\S+)
+
+Start
+  ^VRF:${VRF}
+  ^PIM
+  ^RPF
+  ^\s+M\s-
+  ^Flags
+  ^\s{4}[RWIHZANTVF]\s-
+  ^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}) -> Continue.Record
+  ^${MULTICAST_GROUP}
+  ^\s+[RWIHZANTVF]\s-
+  ^(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}) -> Continue.Record
+  ^${MULTICAST_GROUP}
+  ^\s+(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}) -> Continue.Record
```

Comment:
Consolidate by making the leading whitespace optional since the action for both really similar lines is Continue.Record

```suggestion
  ^${MULTICAST_GROUP}
  ^\s+?(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}) -> Continue.Record
```

Reply:
Yeah that was a merge issue as a result of the last commit accepting the proposed suggestion in the GitHub UI. I have reverted all of the changes from that commit, but fixed the literal whitespaces within the lines. Tests are now passing locally